sunpy/coordinates/tests/test_sun.py

```diff
@@ -7,7 +9,7 @@
 from astropy.time import Time
 import astropy.units as u
 from astropy.tests.helper import assert_quantity_allclose
-from astropy.utils.exceptions import ErfaWarning
+from astropy.utils.exceptions import AstropyWarning, ErfaWarning
```

Comment:
Why add `AstropyWarning`?  Is this a vestigial edit?

Reply:
It's a hangover from a change I didn't need to make in the end, will remove and squash the commits.